app/scripts/services/factions.service.js

```diff
@@ -46,7 +46,7 @@
             angular.forEach(character.reputation, function(rep, index) {
                 standing[rep.id] = {
                     level: rep.standing,
-                    perc: (rep.value / rep.max) * 100,
+                    perc: (isNaN((rep.value / rep.max) * 100) ? 100 : (rep.value / rep.max) * 100),
```

Comment:
There's duplication here. Could you use a temporary variable instead?

Reply:
Added a temporary variable.